docs/components/geometry.md

```diff
@@ -6,7 +6,23 @@ parent_section: docs
 section_title: Components
 section_order: 4
 source_code: src/components/geometry.js
-examples: []
+examples: 
+
+ - title:Creating Shapes
+    src: https://glitch.com/edit/#!/ex-2-geometry?path=index.html:1:0
```

Comment:
you can probably remove `?path=index.html:1:0` (or add them to the other URLs)

Reply:
Nope, you need these so they don't route to the README.

Reply:
my question was more about the consistency with the URLs defined below